code/game/gamemodes/clock_cult/clock_effects/servant_blocker.dm

```diff
@@ -14,7 +14,7 @@
 /obj/effect/clockwork/servant_blocker/CanPass(atom/movable/M, turf/target)
 	var/list/target_contents = M.GetAllContents() + M
 	for(var/mob/living/L in target_contents)
-		if(is_servant_of_ratvar(L) && get_dir(M, src) != dir) //Unless we're on the side the arrow is pointing directly away from, no-go
+		if(is_servant_of_ratvar(L) && get_dir(M, src) != dir && L.stat == DEAD && GLOB.ark_of_the_clockwork_justiciar.progress_in_seconds < GATEWAY_REEBE_FOUND) //Unless we're on the side the arrow is pointing directly away from, no-go
```

Comment:
this will make it ONLY prevent dead servants

Reply:
riiight